docs/colors/tokens.scss

```diff
@@ -1,6 +1,6 @@
-@use '~@onfido/castor';
+@use '~@onfido/castor' as theme;
 
 :export {
-  @include castor.day();
-  @include castor.night();
+  @include theme.day();
+  @include theme.night();
```

Comment:
🤓  [no action required]
Consumers would probably use `@include castor.day` so I don't think it's a big deal to have this import not renamed here.
Just thinking out loud, I don't mind either version.